monkey/infection_monkey/exploit/HostExploiter.py

```diff
@@ -73,6 +74,8 @@ def exploit_host(self):
         result = None
         try:
             result = self._exploit_host()
+        except FailedExploitationError as e:
+            logger.debug(e)
```

Comment:
Why not `logger.exception('Failed exploitation')` or something like that?

Reply:
Because the raiser of this error provided the message that best describe the cause

Reply:
I think when you use `logger.exception('something')` it also logs the exception message of the caught exception (it gets it from the context you're in)

Reply:
Because it would log on `ERROR` level and this is not an error, we expect exploiters to fail.

Reply:
then maybe using `logger.debug(exc_info=True)` would be better. That seems to be the proper way to log the exception info, and you can keep the `DEBUG` level